app/liquid_tags/katex_tag.rb

```diff
@@ -2,7 +2,7 @@ class KatexTag < Liquid::Block
   PARTIAL = "liquids/katex".freeze
   KATEX_EXISTED = "katex_existed".freeze
 
-  def initialize(tag_name, markup, tokens)
+  def initialize(_tag_name, markup, _parse_context)
```

Comment:
I think that `markup` could benefit from being written as `_markup` here since it seems unused - thoughts? 😄

Reply:
@juliannatetreault Ah but it is being used [here](https://github.com/thepracticaldev/dev.to/blob/7930a1a784d975a014d2160d436fae21af3ac780/app/liquid_tags/katex_tag.rb#L38); it's just a little obfuscated since it is [initialized by the gem itself](https://github.com/Shopify/liquid/blob/1feaa6381300d56e2c71b49ad8fee0d4b625147b/lib/liquid/tag.rb#L29)

Reply:
Ohh, I see! Thanks for pointing that out.